pom.xml

```diff
@@ -517,10 +529,11 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
             <version>2.22.1</version>
-            <configuration>
-                <!-- see: https://stackoverflow.com/questions/53010200/maven-surefire-could-not-find-forkedbooter-class -->
-                <argLine>-Xmx1024m -Xms1024m -XX:MaxPermSize=512m -Djdk.net.URLClassPath.disableClassPathURLCheck=true</argLine>
-            </configuration>
+<!--        this causes the sonar scan to fail-->
```

Comment:
Maybe just disable it in the sonar profile?

Reply:
Won't this affect sonar testing coverage ?